tests/utils/staking.go

```diff
@@ -371,6 +371,132 @@ func completeValidatorRegistration(
 	return SendTransactionAndWaitForSuccess(context.Background(), subnet, signedRegistrationTx)
 }
 
+func InitializeAndCompleteNativeValidatorRegistration(
```

Comment:
For completeness, can we add `InitializeAndCompleteERC20ValidatorRegistration` and call that from `erc20_token_staking.go`?